http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -154,8 +155,9 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
             Some(createRuntimeResponse.asyncRuntimeFields),
             ctx.now
           )
+
         // Save the VM image and async fields in the database
-        val clusterImage = RuntimeImage(RuntimeImageType.VM, createRuntimeResponse.customImage.asString, ctx.now)
+        val clusterImage = RuntimeImage(RuntimeImageType.VM, createRuntimeResponse.customImage.asString, None, ctx.now)
```

Comment:
is there a reason why we save the `homeDirectory` as `None` here and not the directory?

Reply:
`homeDirectory` is really only used for `jupyter` (because we need it for setting up jupyter container)...we don't really care about home directory for other images

Reply:
ahhh okay makes sense